addons/medical_treatment/functions/fnc_treatment.sqf

```diff
@@ -69,7 +69,7 @@ if (IS_UNCONSCIOUS(_target) && EGVAR(medical,allowUnconsciousAnimationOnTreatmen
 
 private _isSelf = _caller isEqualTo _target;
 
-if (!_isSelf && {vehicle _target == _target} && {_patientAnim != ""}) then {
+if (!_isSelf && {vehicle _target == _target} && {_patientAnim != ""} && {alive _target}) then {
```

Comment:
we should probably do an `isAwake` check instead.

Reply:
May only apply to actually dead bodies, not ragdolls?

Reply:
To be honest I've never actually noticed treatment animations before 😄 Maybe that means they're doing their job though

I'm in favour of an isAwake check 👍

Reply:
Treatment animations have always been disabled by default iirc. Due to some very valid reasons documented in a pr way back. Or maybe just slack?